static/styles/courses/course.scss

```diff
@@ -247,6 +247,11 @@
 			border-radius: 5px;
 			outline: none;
 		}
+
+		.row > div,
+		.row > h2 {
+			max-width: 1140px;
```

Comment:
Just as I'm curious: do we have defined that somewhere as a variable for max content width?

Reply:
nope, I took that from the bootstrap page as the width value for XL, so the biggest width that is supported for their variables